### PR TITLE
using tags

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Does NOT need to go ASAP

### CODE REVIEW

**Comment by mjoyce91** [comment]
>./src/actions/notifications.js
  Line 4:10:  'get' is defined but never used  no-unused-vars

File: src/actions/notifications.js [hunk 1]
```diff
@@ -236,15 +235,14 @@ export function markNotifications({ ids = new Set(), markAsRead = false, shouldD
...
       cancelToken: new CancelToken((c) => {
         cancelRanking = c;
       }),
     })
       .then(({ data }) => {
-        const data$ = data.results.filter(a => (get(a, 'tags') || []).includes('handshake_bidder'));
-        const ids = data$.map(b => b.id);
-        data$.forEach(n => {
+        const ids = data.map(b => b.id);
+        data.forEach(n => {
```
Comment (mjoyce91) [inline src/actions/notifications.js:245]:
```js
const data$ = get(data, 'results', [])
const ids = data$.map(b => b.id);
data$.forEach(n => {
```

Comment (elizabeth-jimenez) [reply src/actions/notifications.js:245]: thanks Mike!

**Comment by mjoyce91** [comment]
import `get`

**Comment by elizabeth-jimenez** [comment]
> import `get`

Ah, the joys of editing on GitHub. Sorry about that; Done!

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**